compiler/src/dotty/tools/dotc/transform/phantom/PhantomTypeErasure.scala

```diff
@@ -0,0 +1,55 @@
+package dotty.tools.dotc.transform.phantom
+
+import dotty.tools.dotc.ast.tpd
+import dotty.tools.dotc.core.Constants.Constant
+import dotty.tools.dotc.core.Contexts._
+import dotty.tools.dotc.core.DenotTransformers._
+import dotty.tools.dotc.core.Symbols._
+import dotty.tools.dotc.core.Types._
+import dotty.tools.dotc.transform.TreeTransforms.{MiniPhaseTransform, TransformerInfo}
+
+class PhantomTypeErasure extends MiniPhaseTransform with InfoTransformer {
```

Comment:
Aren't we supposed to also erase values and parameters of phantom types here? Without a leading doc comment, impossible to tell.